docs/plugins/event-list.md

```diff
@@ -0,0 +1,55 @@
+---
+id: EventList
+title: List of Plugin Events
+sidebar_label: List of Plugin Events
+slug: /plugins/EventList
+---
+
+
+# List of Plugin Events
+
+- [Create Server](plugins\plugin-events\create-server.md)
+
+
+- [Create Package Json](plugins\plugin-events\create-package-json.md)
+
+
+- Resources 
+
+  - [Create Entity Service](plugins\plugin-events\create-entity-service.md)
+  - [Create Entity Service Base](plugins\plugin-events\create-entity-service-base.md)
+
+  - [Create Entity Controller](plugins\plugin-events\create-entity-controller.md)
+  - [Create Entity Controller Base](plugins\plugin-events\create-entity-controller-base.md)
+
+  - [Create Entity Resolver](docs\plugins\plugin-events\create-entity-resolver.md)
+  - [Create Entity Resolver Base](docs\plugins\plugin-events\create-entity-resolver.md)
+
+  - [Create Entity Controller Spec](docs\plugins\plugin-events\create-entity-controller-spec.md)
+
+
+- Create Entity DTO 
+
+- [Create Swagger](docs\plugins\plugin-events\create-swagger.md)
+
+- Create Server App Module
+
+- [Create Seed](docs\plugins\plugin-events\create-seed.md)
+
+- [Create Server Auth](docs\plugins\plugin-events\create-server-auth.md)
+
+- Create Entity DTO 
+
+- [Create Message Broker](plugins\plugin-events\create-message-broker.md)
+   - [Create Message Broker Topic Enum](plugins\plugin-events\create-message-broker-topics-enum.md)
+   - [Create Message Broker Client Options Factory](plugins\plugin-events\create-message-broker-client-options-factory.md)
+   - [Create Message Broker NestJS Module](plugins\plugin-events\create-message-broker-nestjs-module.md)
+   - [Create Message Broker Service Base](plugins\plugin-events\create-message-broker-service-base.md)
+   
+- [Create Server Auth](docs\plugins\plugin-events\create-server-auth.md)
+- [Create Prisma Schema](plugins\plugin-events\create-prisma-schema.md)
+- [Create Server Dot Env](plugins\plugin-events\create-server-dot-env.md)
+- [Create Server Docker Compose](plugins\plugin-events\create-server-docker-compose.md)
+- Create Server Docker Compose DB
```

Comment:
Why is this one not a link? we have this event:
https://github.com/amplication/amplication/issues/4112